sync/sync.go

```diff
@@ -358,8 +362,28 @@ Register:
 	}
 
 	logrus.Infof("reloadAndVerify %v", replicaAddress)
-	return t.reloadAndVerify(replicaAddress, toClient)
+	if err := t.reloadAndVerify(replicaAddress, toClient); err != nil {
+		return err
+	}
 
+	t.preload(s)
+	return nil
+}
+
+func (t *Task) preload(s *replica.Server) {
+	logrus.Info("Start reading extents")
+	if s.PreloadStatus == "Started" {
```

Comment:
looks good.. will review it keenly

Reply:
As mentioned in one of above comment, PreloadStatus need to be set to 'Started' right after this check and before go routine is started.

Reply:
Sounds good